src/PrestaShopBundle/Translation/TranslatorLanguageLoader.php

```diff
@@ -73,21 +93,64 @@ public function loadLanguage(TranslatorInterface $translator, $locale, $withDB =
         }
         $translator->addLoader('xlf', new XliffFileLoader());
 
+        // Load the theme translations catalogue
+        $finder = Finder::create()
+            ->files()
+            ->name('*.' . $locale . '.xlf')
+            ->notName($this->isAdminContext ? '^Shop*' : '^Admin*')
+            ->in($this->getTranslationResourcesDirectories($theme));
+
+        foreach ($finder as $file) {
+            list($domain, $locale, $format) = explode('.', $file->getBasename(), 3);
+            $translator->addResource($format, $file, $locale, $domain);
+            if ($withDB) {
+                $translator->addResource('db', $domain . '.' . $locale . '.db', $locale, $domain);
+            }
+        }
+
+        // Load modules translation catalogues
+        if (defined(_DB_PREFIX_)) { // if not defined, no need to go further, we won't be able to get the active modules
```

Comment:
If you use the Kernel repository method you won't get any errors because they are already handle by a try catch.